libcomps.spec

```diff
@@ -51,36 +37,16 @@ BuildRequires:  doxygen
 %description doc
 Documentation files for libcomps library.
 
-%package -n python-%{name}-doc
+%package -n python3-%{name}-doc
```

Comment:
It's unclear to me whether this is actually appropriate, since on Fedora there seems to be more packages following the pattern ```python-*-doc``` than ```python3-*-doc```, although plenty examples of both conventions exist.

Reply:
I can revert it if it's not

Reply:
Or if it's not worth doing *right now*

Reply:
Not sure about the naming convention, I'd guess it also depends on whether the documentation is (was) shared for both python versions.

I don't think we can just rename the package, we would need to add an obsolete to provide an upgrade path. Please leave the old package name, if we do the rename, it should be done separately.

Reply:
Reverted